firefox-ios/Client/Frontend/PasswordManagement/PasswordManagerOnboardingViewController.swift

```diff
@@ -56,62 +62,91 @@ class PasswordManagerOnboardingViewController: SettingsViewController {
     }
 
     override func viewDidLoad() {
-        super.viewDidLoad()
+            super.viewDidLoad()
+
+            self.title = .Settings.Passwords.Title
+
+            self.view.addSubviews(scrollView)
+
+            NSLayoutConstraint.activate([
+                scrollView.leadingAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.leadingAnchor),
+                scrollView.trailingAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.trailingAnchor),
+                scrollView.topAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.topAnchor),
+                scrollView.bottomAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.bottomAnchor),
+            ])
+            scrollView.addSubview(contentView)
+            NSLayoutConstraint.activate([
+                contentView.widthAnchor.constraint(equalTo: scrollView.frameLayoutGuide.widthAnchor),
+                contentView.topAnchor.constraint(equalTo: scrollView.contentLayoutGuide.topAnchor),
+                contentView.bottomAnchor.constraint(greaterThanOrEqualTo: scrollView.bottomAnchor),
+                contentView.heightAnchor.constraint(greaterThanOrEqualTo: scrollView.heightAnchor)
+            ])
+            contentView.addSubviews(onboardingMessageLabel, learnMoreButton, continueButton)
+            configureContentViewChildConstraints()
+    }
 
-        self.title = .Settings.Passwords.Title
+    @objc
+    func learnMoreButtonTapped(_ sender: UIButton) {
+        let viewController = SettingsContentViewController(windowUUID: windowUUID)
+        viewController.url = SupportUtils.URLForTopic("set-passcode-and-touch-id-firefox")
+        navigationController?.pushViewController(viewController, animated: true)
+    }
 
-        self.view.addSubviews(onboardingMessageLabel, learnMoreButton, continueButton)
+    @objc
+    func proceedButtonTapped(_ sender: UIButton) {
+        continueFromOnboarding()
+    }
 
+    private func configureContentViewChildConstraints() {
         NSLayoutConstraint.activate(
             [
                 onboardingMessageLabel.leadingAnchor.constraint(
-                    equalTo: self.view.safeAreaLayoutGuide.leadingAnchor,
+                    equalTo: contentView.safeAreaLayoutGuide.leadingAnchor,
```

Comment:
Nit: Using `safeAreaLayoutGuide` is not necessary for all the constraints in this method as we view that is constraint against the edges of the view controllers view (the `scrollView`) already uses it.